ui/pages/home/home.component.js

```diff
@@ -734,6 +742,37 @@ export default class Home extends PureComponent {
     }
     const tabPadding = process.env.MULTICHAIN ? 4 : 0; // TODO: Remove tabPadding and add paddingTop={4} to parent container Box of Tabs
 
+    const networkPicker = process.env.MULTICHAIN ? (
+      <Box marginTop={4} marginInlineEnd={4}>
+        <PickerNetwork
+          className="home__tab__network-picker"
+          avatarNetworkProps={{
+            backgroundColor: testNetworkBackgroundColor,
+          }}
+          margin={2}
+          label={currentNetwork?.nickname}
+          src={currentNetwork?.rpcPrefs?.imageUrl}
+          onClick={(e) => {
+            e.stopPropagation();
+            e.preventDefault();
```

Comment:
Optional: can we move these 2 events related actions to component `PickerNetwork` level?